x/oracle/simulations/genesis.go

```diff
@@ -0,0 +1,129 @@
+package simulation
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// Simulation parameter constants
+const (
+	votePeriodKey               = "vote_period"
+	voteThresholdKey            = "vote_threshold"
+	rewardBandKey               = "reward_band"
+	rewardDistributionWindowKey = "reward_distribution_window"
+	slashFractionKey            = "slash_fraction"
+	slashWindowKey              = "slash_window"
+	minValidPerWindowKey        = "min_valid_per_window"
+)
+
+// GenVotePeriod randomized VotePeriod
```

Comment:
Can we add to the godocs for each of these of what value ranges they generate?

e.g. `// GenVotePeriod produces a randomized VotePeriod in the range of [x,y]`